CLIrefactor.py

```diff
@@ -7,104 +7,115 @@
 import loadgame
 import sys
 import os
+import argparse
 
-#create controller object
-'''
-controller = ctrl.controller()
-print("Hello! Welcome to MediaTek's Spelling bee!")
-print("The goal of this game is to guess as many words as you can to accumulate points!")
-controller.ensureYesOrNo()'''
-class view:
-    #while loop for the CLI.
-    def __init__(self):
-        self.controller = ctrl.controller()
+parser = argparse.ArgumentParser(description='Spelling Bee Game')
+parser.add_argument('--cli', action='store_true', help='run in command-line mode')
+args = parser.parse_args()
 
-    def startGame(self):
-        self.controller.ensureYesOrNo()
-        print('''
+if args.cli:
```

Comment:
Instead of putting this all under the if state for this, couldnt you put it below under the class. refer to comments are bottom of file for this